src/web/WebSchedulerService.cpp

```diff
@@ -136,6 +136,7 @@ bool WebSchedulerService::get_value_info(JsonObject output, const char * cmd) {
     if (Helpers::toLower(cmd) == F_(commands)) {
         output[F_(info)]     = Helpers::translated_word(FL_(info_cmd));
         output[F_(commands)] = Helpers::translated_word(FL_(commands_cmd));
+        output[F_(values)]   = Helpers::translated_word(FL_(values_cmd));
```

Comment:
same as before hidden or not?

Reply:
this is a special case as `info` shows the same data as `values` in the scheduler right now.

Reply:
yes, also for custom. If needed we can add more attributes to info.